tests/chainerx_tests/unit_tests/routines_tests/test_indexing.py

```diff
@@ -101,15 +103,24 @@
     ((2, 3, 4),
      (1, slice(2,), chainerx.newaxis, slice(1, 3), chainerx.newaxis)),
 ])
-def test_getitem(xp, shape, indices):
-    a = array_utils.create_dummy_ndarray(xp, shape, 'int32')
-    return a[indices]
+class TestGetitem(op_utils.NumpyOpTest):
+    # TODO(niboshi): Remove this
+    check_numpy_strides_compliance = False
+
+    def generate_inputs(self):
+        x = array_utils.create_dummy_ndarray(numpy, self.shape, 'float32')
```

Comment:
How about getting rid of the `array_utils` dependency in this file by generating inputs similar to `TestTake` below? You should be able to use the `contiguous` attribute for testing "padded" arrays.

Reply:
Fixed.

>You should be able to use the contiguous attribute for testing "padded" arrays.

Input arrays are non-contiguous by default.